splicemachine/ml/utilities.py

```diff
@@ -175,7 +175,7 @@ def plotROC(self, trainingSummary, ax):
         """
         Plots the receiver operating characteristic curve for the trained classifier
 
-        :param trainingSummary: TrainingSummary object accessed by .summary after fitting a binary classification object
+        :param trainingSummary: TrainingSummary object accessed by .summary after fitting a binary classification object ONLY VALID FOR LOGISTIC REGRESSION
```

Comment:
Change this function to take in a model, and check that the model is a logistic regression -- throw an assertion error if it's not. If it is, get the summary within the function